src/status_im/contexts/onboarding/events.cljs

```diff
@@ -67,6 +67,28 @@
  (fn [_ [error]]
    {:dispatch [:biometric/show-message (ex-cause error)]}))
 
+;; Navigate to next screen once onboarding animation and account creation both are complete
+;; Note: If account creation fails, user will get stuck on loading screen
+;; TODO: Show error screen in case account creation/restoration fails
+(rf/defn account-creation-complete
```

Comment:
@Parveshdhull we lose linting when this macro is used. If this event is not being used in places with `rf/merge` we should just use `reg-event-fx`.

Reply:
Thank you for reviewing the PR, updated